tendermint/src/node/info.rs

```diff
@@ -82,7 +76,7 @@ pub struct OtherInfo {
     pub tx_index: TxIndexStatus,
 
     /// RPC address
-    pub rpc_address: net::Address,
+    pub rpc_address: String,
```

Comment:
One unresolved issue is whether using `String` instead of `net::Address` for `rpc_address` will break any functionality. The main difference is in the implementations of `Deserialize` and `Display`. Any idea of how this field is used in practice?

Reply:
@thanethomson Might have some input here.

Reply:
No idea here actually.